src/generator/exercises/acronym.cr

```diff
@@ -29,6 +29,6 @@ class AcronymTestCase < ExerciseTestCase
   end
 
   def test_name
-    description
+    "does #{description}"
```

Comment:
Cool and then we can add in whatever verbiage to make the test case descriptions sensical.